ui/selectors/selectors.js

```diff
@@ -1998,6 +1998,11 @@ export function getShouldShowAggregatedBalancePopover(state) {
   return shouldShowAggregatedBalancePopover;
 }
 
+export function getDismissSmartAccountSuggestionEnabled(state) {
```

Comment:
Should we use `ui/pages/confirmations/selectors/preferences.ts` instead of adding to this huge file?

Reply:
fixed